components/dropdown/demo/api.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable jsdoc/require-jsdoc, no-magic-numbers */
 import { showExample } from './../apiExamples/programmaticallyShow.js';
 import { hideExample } from './../apiExamples/programmaticallyHide.js';
+import './index.js';
```

Comment:
`./index.js` also has `initExamples` function which does exactly the same. It might be good to utilize that.